packages/node/package.json

```diff
@@ -75,6 +75,13 @@
     "LICENSE",
     "README.md"
   ],
+  "jest": {
+    "preset": "ts-jest",
+    "testEnvironment": "node",
+    "moduleNameMapper": {
+      "axios": "axios/dist/node/axios.cjs"
+    }
+  },
```

Comment:
Jest v27 and Axios latest version have a conflict as Jest is not importing the CJS version of Axios